### PR TITLE
Functional elm app. Does not hit front end yet

### PR DESCRIPTION
**Pull request by nqthqn**
Deps
 - bulma.css
 - font awesome

screen shot 2017-02-05 at 4 26 30 pm

### CODE REVIEW

**Comment by nqthqn** [comment]
Oops, ignored elm-stuff. Might as well ignore bundle! One mo...

**Comment by daved** [comment]
>Oops, ignored elm-stuff. Might as well ignore bundle! One mo...

These sort of paths/files should be in the global gitignore.  A lot of tutorials have the global ignore file as ~/.gitignore_global, but I recently found out that the true default is ~/.config/git/ignore.

Please remove the .gitignore from this branch.

Edit to add - We do want to check-in the generated code (bundle.js).  There are different dependency scopes, and checking in the generated JS allows Go development to continue properly without having Elm as a dep.

File: internal/spa/src/srv/assets/css/elm-package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (daved) [inline internal/spa/src/srv/assets/css/elm-package.json:1]: This file seems to be useless.  It should probably be removed.

**Comment by daved** [comment]
~~This is a fussy request, but I favor using reserved top level domains for dev/docs.  https://tools.ietf.org/html/rfc2606#section-2~~

~~https://www.site.example, you@yourdomain.example, etc.~~

~~That's also why I use .localhost for local development.  We can be sure the domains won't work apart from our purposes.~~

Second level domain is reserved...
https://tools.ietf.org/html/rfc2606#section-3

File: internal/spa/src/srv/assets/css/elm-package.json (hunk 1 above)
Comment (nqthqn) [reply internal/spa/src/srv/assets/css/elm-package.json:1]: oops

**Comment by nqthqn** [comment]
"Automatically created by GitHub for Mac" was in my global ignore :/ hmff...

**Comment by daved** [comment]
Merged manually.

*Dropped commit c3baac58d0d06d3274db5e96b0580cf975c8900f.
*Amended message of commit 50a3cf64803fbd25aff3ee61c145071a15c5d72f.

**Pull request closed without merging**